doghouse/server/storage.go

```diff
@@ -0,0 +1,58 @@
+package server
+
+import (
+	"context"
+
+	"google.golang.org/appengine/datastore"
+)
+
+type Installation struct {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type Installation should have comment or be unexported